actions/notification_actions.jsx

```diff
@@ -68,17 +68,21 @@ export function sendDesktopNotification(post, msgProps) {
         const member = getMyChannelMember(state, post.channel_id);
         const isCrtReply = isCollapsedThreadsEnabled(state) && post.root_id !== '';
 
-        if (!member || isChannelMuted(member) || userStatus === UserStatuses.DND || userStatus === UserStatuses.OUT_OF_OFFICE) {
+        if (!member || (!persistent && isChannelMuted(member)) || userStatus === UserStatuses.DND || userStatus === UserStatuses.OUT_OF_OFFICE) {
             return;
         }
 
-        let notifyLevel = member?.notify_props?.desktop || NotificationLevels.DEFAULT;
+        // we are bypassing channel member's notify props in the case of persistent notifications
+        let notifyLevel = (!persistent && member?.notify_props?.desktop) || NotificationLevels.DEFAULT;
```

Comment:
0/5 Similarly, bypassing the desktop notification setting seems odd. If a user really wanted to turn off desktop notifications, they could still get around it by disabling them at an OS level

Reply:
It's only bypassing the channel specific `notify_prop` for desktop notifications.
User `notify_props` are respected.

cc @esethna